src/native/corehost/apphost/common_controls.manifest

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
+  <dependency>
```

Comment:
Maybe for size reasons we could remove all of the whitespace here - not sure if it will help though.

Reply:
You may be able to remove the xml declaration line as well.